src/main/java/co/lotc/lever/cmd/ShowItem.java

```diff
@@ -18,9 +18,11 @@ public void invoke(Persona source, Player target) {
 		validate(LocationUtil.isClose(source.getPlayer(), target, 16), "Can only show items to nearby players!");
 		ChatBuilder b = new ChatBuilder(source.getName()).color(GOLD)
 				.append(" is showing you ").color(AQUA)
-				.append('[' + ItemUtil.getDisplayName(is) + ']').color(WHITE).hoverItem(is);
-		
-		b.send(target);
+				.append('[').color(WHITE)
+				.append(ItemUtil.getDisplayName(is)).hoverItem(is)
```

Comment:
Still need to convert legacy text to chat components here

Reply:
idk how to do that :)